### DISCUSSION TITLE
Command to create event from text selection

### DISCUSSION THREAD

**Discussion by thePHTest** [davish/obsidian-full-calendar#398] [Feature Requests]
I would like to add a command that can take the current text selection for making a new event with that text as the title or description. And then once the event is created, it can put a link inline with the selected task to the new event.

This would greatly help me create events from my master todo list and see which ones are scheduled or not. Would this run into issues with links breaking or being stale when rescheduling or removing events however?

I am down to help with this issue if someone can point me in the right direction in the codebase. I just haven't developed Obsidian plugins in particular before. I am also fairly new to using Obsidian but love it so far. Thanks for this plugin!

**Comment by AthreyaV** [comment]
This should be possible using QuickAdd and Templater, using a capture command to feed the highlighted text into a new file where the calendar events are stored
